README.md

```diff
@@ -161,6 +314,250 @@ more metric, you must run CASTanet for the first metric
 and then run CASTanet subsequently for
 each additional metric (with the corresponding CLI command).
 
+### CASTanet as a Python Library
+
+CASTanet is also available on PyPI to be used as a Python
+library. Find it [here](https://pypi.org/project/castanet/).
+With the CASTanet library, a user is able to investigate
+their Python files with many different function calls.
+Specifically, CASTanet is broken down into two parts:
+
+1. `generate_trees`: Traverses a directory and generates
+concrete-abstract-syntax trees of Python files using
+LibCST
+1. `counter`: Uses concrete-abstract-syntax-trees to
```

Comment:
LibCST is so useful because it creates a "compromise between an abstract syntax tree (AST) and a concrete syntax tree (CST), therefore a merge of the two of them can be referred to as a concrete abstract syntax tree (CAST). This has been clarified in documentation.